apps/saleor-app-checkout/setupTestsAfterEnv.ts

```diff
@@ -18,6 +18,7 @@ afterEach(() => server.resetHandlers());
 afterEach(() => {
   consoleTypes.forEach((type) => {
     // @ts-expect-error
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
     console[type]?.mockClear?.();
```

Comment:
This is more safe, because we're not making an unsafe `?.()` call. Optional chaining function call is not actually checking if the property is a function before calling it, instead it only checks if the value is not `undefined` or `null`. This means if, for some reason, it's something else (e.g. a string / an object) then this call will throw. I dunno why that's the spec, it makes no sense 🤦🏻‍♂️ 

```suggestion
    typeof console[type]?.mockClear === "function" && console[type].mockClear();
```